semgrep/semgrep/constants.py

```diff
@@ -38,7 +38,7 @@ class OutputFormat(Enum):
     VIM = auto()
 
     def is_json(self) -> bool:
-        return self == OutputFormat.JSON or self == OutputFormat.JSON_DEBUG
+        return self in [OutputFormat.JSON, OutputFormat.JSON_DEBUG, OutputFormat.SARIF]
```

Comment:
This change isn't strictly necessary, but I'm hoping to also use this function in `tests.conftest._run_semgrep` to detect all JSON output formats.